Meshery-Adapters/istio-meshery-adapter/step2.md

```diff
@@ -1,10 +1,10 @@
 Meshery allows you to install any service mesh with a single click:
 
-## 1. Select `Istio` from the Management menu
+## 1. Select `Istio` from the Performance menu
```

Comment:
We find `Istio` on the `Lifecycle` Menu and not the `Performance` menu, please change this

Reply:
Will make the required changes

> @KerinPithawala The picture which you changed too doesn't correctly show where `Istio` can be found. Please update the picture with a red box on `Isitio` which can be found on the `Lifecycle` Menu

I will take new picture according to this review.